sig/datadog/core/environment/cgroup.rbs

```diff
@@ -0,0 +1,19 @@
+module Datadog
+  module Core
+    module Environment
+      # Reads information from Linux cgroups.
+      # This information is used to extract information
+      # about the current Linux container identity.
+      # @see https://man7.org/linux/man-pages/man7/cgroups.7.html
```

Comment:
I suggest stripping all comments from the .rbs files, to avoid them getting out of sync with the .rb files, since after the initial creation of the .rbs, we're supposed to be maintaining them manually.

Reply:
I wish rbs was a bit further along in this... I'm getting more inclined to switch to this, but I really dislike how it leaves so much up to the user to solve...